src/test/java/com/microsoft/sqlserver/testframework/Utils.java

```diff
@@ -8,13 +8,50 @@
 
 package com.microsoft.sqlserver.testframework;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 /**
  * Generic Utility class which we can access by test classes.
  * 
  * @since 6.1.2
  */
 public class Utils {
+    public static final Logger log = Logger.getLogger("Utils");
 
+    // 'SQL' represents SQL Server, while 'SQLAzure' represents SQL Azure. 
+    private static String _serverType         = null;  
+    public  static final  String  SERVER_TYPE_SQL_SERVER = "SQL";
+    public  static final  String  SERVER_TYPE_SQL_AZURE  = "SQLAzure";
+    
+    public static String getServertype()
+    {
+        if (_serverType == null)
+        {
+            String serverTypeProperty = getConfiguredProperty("server.type");
+            if (serverTypeProperty == null)
+            {
+                // default to SQL Server
+                _serverType = SERVER_TYPE_SQL_SERVER;
+            }
+            else if (serverTypeProperty.equalsIgnoreCase(SERVER_TYPE_SQL_AZURE))
```

Comment:
Like to have constant on left hand side for comparison. Although it's good that you are checking serverTypeProperty is null nor not before any comparison.